README.md

```diff
@@ -1 +1,24 @@
-# hw01
\ No newline at end of file
+# About Me
+Hello, my name is Tianyi Qian.  
+
+### Educational Background  
+
+---
+
+I'm a grad student in the MAPSS program at the University of Chicago and I concentrate in **Quantitative Methods and Social Anlaysis**. I studied **Sociology** and **Media Studies** as an undergrad. I'm hoping to learn the following from this class:
+
+
+* _data manipulation_
+* _data visualization_
+* _get more comfortable with programming in R_
+
+### Links
+
+---
+
+Here's my [github page](https://github.com/VivianQian19)
+
+![](Hello.png)
```

Comment:
Merit: The use of links and photos are great. 
Improvement: You can try to link the photo from a website.